bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
+    - By asking your DataRescue guide or by filling out [this form](XXX)
```

Comment:
"...or by filling out **the form that your DataRescue guide points you to." or similar?